### PR TITLE
feat: add serve command in workflow

### PR DESCRIPTION
**Pull request by AkashKumar7902**
## Related Issue
  - Info about Issue or bug

Closes: #1200 

#### Describe the changes you've made
added serve command to be ran in workflow. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by AkashKumar7902** [comment]
@shivamsouravjha PTAL!

File: .github/workflows/python-linux.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ sudo rm -rf keploy/
 
 # Update the global noise to ignore the Allow header.
 config_file="./keploy-config.yaml"
-sed -i 's/"header": {}/"header":{"Allow":[]}/' "$config_file"
+sed -i 's/header: {}/header: {"Allow":[]}/' "$config_file"
 
 # Wait for 5 seconds for it to complete
```
Comment (shivamsouravjha) [inline .github/workflows/python-linux.sh:30]: why these changes?

File: .github/workflows/node-linux.sh [hunk 2]
```diff
@@ -63,14 +63,20 @@ done
 # Start keploy in test mode.
 sudo -E env PATH=$PATH ./../../keployv2 test -c 'node src/app.js' --delay 10
```
Comment (shivamsouravjha) [inline .github/workflows/node-linux.sh:65]: instead of node src/app.js you can replace it with ```npm start --prefix /src/app.js`` in both test and record

File: .github/workflows/python-linux.sh (hunk 1 above)
Comment (AkashKumar7902) [reply .github/workflows/python-linux.sh:30]: I made these changes to solve #1210 , I will be reverting this change and make a separate PR.

File: .github/workflows/node-linux.sh (hunk 2 above)
Comment (AkashKumar7902) [reply .github/workflows/node-linux.sh]: done!

**Review by shivamsouravjha — APPROVED** [review]
LGTM
